cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -45,18 +45,30 @@
     "end tag": "xmlEndTag",
 }
 
+select_multiple_modifiers = {
+    "every", 
+    "all"
+}
+
 
-@mod.capture(rule="[every] {user.cursorless_scope_type}")
+@mod.capture(rule="[(every|all)] {user.cursorless_scope_type}")
 def cursorless_containing_scope(m) -> dict[str, dict[str, Any]]:
-    """Expand to containing scope"""
-    return {
-        "modifier": {
-            "type": "containingScope",
-            "scopeType": m.cursorless_scope_type,
-            "includeSiblings": m[0] == "every",
+    """Expand to every scope"""
```

Comment:
Definitely open to putting this into another file, my python is a little weak as is my understanding of how it interacts with Talon. Just lmk.